samples/snippets/src/main/java/com/example/storage/ConfigureRetries.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.storage;
+// [START storage_configure_retries]
+
+import com.google.api.gax.retrying.RetrySettings;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.StorageRetryStrategy;
+import org.threeten.bp.Duration;
+
+public final class ConfigureRetries {
+  public static void main(String[] args) {
+    String bucketName = "my-bucket";
+    String blobName = "blob/to/delete";
+    deleteBlob(bucketName, blobName);
+  }
+
+  static void deleteBlob(String bucketName, String blobName) {
+    // Update the retry settings
+    RetrySettings retrySettings =
+        StorageOptions.getDefaultRetrySettings()
+            .toBuilder()
+            // to set the max number of attempts to 10
+            .setMaxAttempts(10)
+            // to set the backoff multiplier to 3.0
+            .setRetryDelayMultiplier(3.0)
+            // to set the max duration of all attempts to 5 minutes
```

Comment:
Seems unrealistically long for a delete, but fine if this is what other langs have.